java/core/src/java/org/apache/orc/OrcConf.java

```diff
@@ -105,6 +105,11 @@
           "(default 10000 rows) else dictionary check will happen before\n" +
           "writing first stripe. In both cases, the decision to use\n" +
           "dictionary or not will be retained thereafter."),
+  DICTIONARY_IMPL("orc.dictionary.implementation", null,
```

Comment:
hash option missing?

Reply:
the value for this config is being processed `createDict`.

Reply:
I guess what I meant was that we should explicitly list the choices (similar to BLOOM_FILTER_WRITE_VERSION):
* rbTree
* hash (not yet implemented

Moreover, I would set the hiveConfName same as the attribute -- I believe null means that it does not apply to Hive

Reply:
Thanks, updated the description in a way similar to `BLOOM_FILTER_WRITE_VERSION`.